### DISCUSSION TITLE
Runner Scale Sets not supported in Github Enterprise Server until v3.9

### DISCUSSION THREAD

**Discussion by jb-2020** [actions/actions-runner-controller#2853] [FAQs] [answered]
While attempting to migrate to the official _Runner Scale Set_ charts the following error was observed: 

```
ERROR Reconciler error {"controller": "autoscalingrunnerset", "controllerGroup": "actions.github.com", "controllerKind": "AutoscalingRunnerSet", "AutoscalingRunnerSet": {"name":"<redacted>","namespace":"<redacted>"}, "namespace": "<redacted>", "name": "<redacted>", "reconcileID": "<redacted>", "error": "404 - had issue communicating with Actions backend: Request returned status: 404 Not Found"}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
	sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:329
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
	sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:274
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
	sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:235
```

The assumed cause is that our instance of Github Enterprise Server (GHES) is not on v3.9.  

Opening this discussion to hopefully prevent some investigations for users of older versions GHES.  

Ideally, the messaging should be improved and made more clear the supported versions of GHES. 

Ref: https://docs.github.com/en/enterprise-server@3.9/admin/release-notes#github-actions

**Comment by Link-** [comment, marked as answer]
Correct, and it's quite easy to figure it out. If you go to our docs: https://gh.io/arc-docs and switch the version drop-down you will only see GHES 3.9+ in the list. That means support has started since that version and not earlier.

I'm moving this to our FAQs for posterity.

![cs_2023-09-22_13-00-49](https://github.com/actions/actions-runner-controller/assets/568794/f0798f32-faa3-421e-961e-493a76316d02)
